### ISSUE TITLE
Student work is still in repository?

### ISSUE THREAD

**Issue by arthur-e** [OscarFHC/EAS538_2018#15]
Hi @mehajain13 @OscarFHC,

It looks like in Meha's last push, many **submitted** assignments were removed from the Week 3 directory:

```
delete mode 100644 Week3_CLT/assignment2_Fri/bartonerin_370585_3819363_embarto_2.R
delete mode 100644 Week3_CLT/assignment2_Fri/bennettalexander_378613_3844888_awbenn_2.R
...
```

I hadn't realized these were in the repository. Since this repository is public, we might want to:

- [ ] Remove all the old assignment submissions;
- [ ] Delete the Github repositories (Oscar's and my fork);
- [ ] Push the clean repositories back up to Github;

Deleting the Github repositories is necessary because even having deleted these files from the repo, they're accessible in previous versions of the repository.

**Comment by mehajain13** [comment]
Good catch! Yes, these old assignments should definitely be deleted. Thanks!

**Comment by arthur-e** [comment]
Should we git rid of these Final Exam/ Quiz materials, too? You might want to keep these somewhere but if the repository is going to be public, these documents can be found by anyone:

```
Quiz1_Rubric.docx
Quiz3_GradingNotes.txt
Quiz3_OC.docx
```

And the `TakeHomeQuiz` directory...?

```sh
$ ls -l TakeHomeQuiz/
total 136
-rw-rw-r-- 1 arthur arthur  2543 Jan 11 09:01 energy_data.csv
-rw-rw-r-- 1 arthur arthur 30996 Jan 11 09:01 housingdata.csv
-rw-rw-r-- 1 arthur arthur   413 Jan 11 09:01 NRE538_TakeHomeQuiz.R
drwxrwxr-x 3 arthur arthur 12288 Jan 11 09:01 TakeHomeQuiz
-rw-rw-r-- 1 arthur arthur  1176 Jan 11 09:01 TakeHomeQuiz_comments.txt
-rw-rw-r-- 1 arthur arthur 21668 Jan 11 09:01 TakeHome_Quiz.docx
-rw-rw-r-- 1 arthur arthur  2848 Jan 11 09:01 Take_Home_Quiz_Meha.R
-rw-rw-r-- 1 arthur arthur  3017 Jan 11 09:01 Take_Home_Quiz.R
-rw-rw-r-- 1 arthur arthur 20840 Jan 11 09:01 TakeHome_Quiz_rubic_20170329.docx
-rw-rw-r-- 1 arthur arthur 24101 Jan 11 09:01 TakeHome_Quiz_Updated.docx
```

**Comment by arthur-e** [comment]
I partially addressed this in Pull Request #16.

**Comment by mehajain13** [comment]
Hi @arthur-e @OscarFHC I moved all of the quiz/test materials onto my local computer and removed them from the github folder. I'll keep copies of everything on my local desktop so if we realize I took out something we need in github let me know and I can repost
